turbinia/message.py

```diff
@@ -41,18 +41,24 @@ class TurbiniaRequest:
     recipe(dict): Recipe to use when processing this request.
     context(dict): A Dict of context data to be passed around with this request.
     evidence(list): A list of Evidence objects.
+    group_name (str): Name for grouping evidence.
+    reason (str): Reason or justification for Turbinia requests.
+    all_args (str): a string of commandline arguments provided to run client.
   """
 
   def __init__(
       self, request_id=None, group_id=None, requester=None, recipe=None,
-      context=None, evidence_=None):
+      context=None, evidence=None, group_name=None, reason=None, all_args=None):
     """Initialization for TurbiniaRequest."""
     self.request_id = request_id if request_id else uuid.uuid4().hex
     self.group_id = group_id if group_id else uuid.uuid4().hex
     self.requester = requester if requester else 'user_unspecified'
     self.recipe = recipe if recipe else {'globals': {}}
     self.context = context if context else {}
-    self.evidence = evidence_ if evidence_ else []
+    self.evidence = evidence if evidence else []
+    self.group_name = group_name if group_name else ''
+    self.reason = reason if reason else ''
+    self.all_args = all_args if all_args else []
```

Comment:
Should have this default to an empty string similar to the other string type attributes here?  If this is a different data type here, then I think this will also mean this would get added to datastore/redis as a different data type, which could potentially cause issues elsewhere.

Reply:
Thanks for point it out! Checked other places might have it as list still.